src/expertsystem/amplitude/helicity_decay.py

```diff
@@ -261,7 +261,8 @@ def register_amplitude_coefficient_name(
                 priority_partner_coefficient_suffix,
             ) = self._generate_amplitude_coefficient_couple(graph, node_id)
 
-            if graph.get_node_props(node_id).parity_prefactor is None:
+            node_props = graph.get_node_props(node_id)
+            if node_props is not None and node_props.parity_prefactor is None:
```

Comment:
Will the new code also crash if there are no node props defined or just build something which is incorrect?

Reply:
05ef4cb5f reverts this